index.js

```diff
@@ -185,6 +185,7 @@ class BitbucketScm extends Scm {
      * @param  {String}       config.repoId     Bitbucket repo ID (e.g., "username/repoSlug")
      * @param  {String}       config.token      Admin Oauth2 token for the repo
      * @param  {String}       config.url        url to create for webhook notifications
+     * @param  {String}     config.actions      Actions for the webhook events
```

Comment:
```suggestion
     * @param  {String}       config.actions      Actions for the webhook events
```
small indentation comment